src/components/grid-aware/PartnersAndSponsorsBlock/PartnersAndSponsorsBlock.module.css

```diff
@@ -0,0 +1,67 @@
+/* Main grid */
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
```

Comment:
You can remove this line, since there is only one row in this outer grid. My ThreeParagraphBlock had to play some tricks to get some of the bleed effect working, but because you don't have that here, you can just remove this and use the default value of one, auto-sized row.

```suggestion
```